src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/ResiliencyReportData.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class ResiliencyReportData : IResiliencyReportData
+    {
+        [JsonConverter(typeof(ResiliencyResource))]
+        ResiliencyResource[] resiliencyResourceList;
+
+        /// <summary>
+        /// Creates an instance of ResiliencyReportData
+        /// </summary>
+        public ResiliencyReportData(string customerName, ResiliencyResource[] resiliencyResourceList)
+        {
+            this.CustomerName = customerName;
+            if (resiliencyResourceList == null)
+            {
+                throw new ArgumentNullException(nameof(resiliencyResourceList), $"{nameof(resiliencyResourceList)} cannot be null");
+            }
+            else
+            {
+                this.resiliencyResourceList = resiliencyResourceList;
+            }
+        }
+        /// <summary>
+        /// Customer's name used for the report's cover. This will normally be either customer's Company or simply Customer's name obtained from the subscription.
+        /// </summary>
+        public string CustomerName { get; set; }
+
+        /// <summary>
+        /// Array containing a list of all the resources to be included in the report along with their 
+        /// </summary>
+
+
```

Comment:
nitpik; whitespace gap & finish description

Reply:
Done. Thank you

Reply:
Fixed. Thank you.